src/test_l1_hdmi_cec_driver.c

```diff
@@ -1236,26 +1232,26 @@ void test_hdmicec_hal_l1_setRxCallback_negative ( void )
 
     //Calling API before open, should pass
     result = HdmiCecSetRxCallback(handle, DriverReceiveCallback, (void*)0xDEADBEEF);
-    if (HDMI_CEC_IO_NOT_OPENED != result) { UT_FAIL ("HdmiCecSetRxCallback failed"); } 
+    if (HDMI_CEC_IO_NOT_OPENED != result) { UT_FAIL_FATAL ("HdmiCecSetRxCallback failed"); }
 
     /* Positive result */
     result = HdmiCecOpen (&handle);
     //if init is failed no need to proceed further
-    UT_ASSERT_EQUAL ( result, HDMI_CEC_IO_SUCCESS );
+    UT_ASSERT_EQUAL_FATAL ( result, HDMI_CEC_IO_SUCCESS );
 
     result = HdmiCecSetRxCallback(0, DriverReceiveCallback, (void*)0xDEADBEEF);
-    if (HDMI_CEC_IO_INVALID_HANDLE != result) { HDMICEC_ASSERT_AUTO_TERM(); UT_FAIL ("HdmiCecSetRxCallback failed"); } 
+    if (HDMI_CEC_IO_INVALID_HANDLE != result) { HDMICEC_TERM(); UT_FAIL_FATAL ("HdmiCecSetRxCallback failed"); }
 
     /*calling hdmicec_close should pass */
     result = HdmiCecClose (handle);
-    if (HDMI_CEC_IO_SUCCESS != result) { UT_FAIL ("HdmiCecClose failed"); }
+    if (HDMI_CEC_IO_SUCCESS != result) { UT_FAIL_FATAL ("HdmiCecClose failed"); }
 
     //Calling API after close, should return success
     result = HdmiCecSetRxCallback(handle, DriverReceiveCallback, 0);
-    if (HDMI_CEC_IO_NOT_OPENED != result) { UT_FAIL ("HdmiCecSetRxCallback failed"); } 
+    if (HDMI_CEC_IO_NOT_OPENED != result) { UT_FAIL_FATAL ("HdmiCecSetRxCallback failed"); }
 
     HdmiCecSetRxCallback(0, DriverReceiveCallback, 0);
```

Comment:
is it required again testing with invalid params? also return value is not capture in 'result' param